ros/angel_system_nodes/package.xml

```diff
@@ -17,6 +17,7 @@
   <depend>image_transport</depend>
   <depend>cv_bridge</depend>
   <depend>python3-opencv</depend>
+  <depend>python3-pyaudio</depend>
```

Comment:
I think we'll need to add the pyaudio dependency to the pyproject list. I also got an import error after pip install pyaudio, so I had to do `apt install python3-pyaudio`.

I think we can add that apt install to the Dockerfile here: https://github.com/PTG-Kitware/angel_system/blob/master/docker/workspace-build/Dockerfile#L20

@Purg, is that the right place to install pyaudio?

Reply:
I think I recommended @derkmed to add this here. If this dep is *only* used in the ROS node space, then this is the correct place to add this dep. The pyproject file encapsulates the deps only for the `angel_system` python package at the root level (NOT the `./ros/` stuff).

Reply:
I also forgot to add that we do not need to add an apt install to the dockerfile as this dep is slurped in via the rosdep component of the build that is part of the container. This step effectively performers additional ros-package specific dep declaration (like here).

Reply:
Ah, sorry, ignore me then. I didn't do a new docker container build so maybe that's why I was getting import errors.